processing/src/main/java/org/apache/carbondata/processing/loading/model/CarbonLoadModelBuilder.java

```diff
@@ -426,6 +432,24 @@ private void validateAndSetColumnCompressor(CarbonLoadModel carbonLoadModel)
     }
   }
 
+
+  private void validateAndSetBinaryDecoder(CarbonLoadModel carbonLoadModel)
+      throws InvalidLoadOptionException {
+    try {
+      String binaryDecoder = carbonLoadModel.getBinaryDecoder();
+      if (StringUtils.isBlank(binaryDecoder)) {
+        binaryDecoder = CarbonProperties.getInstance().getProperty(
+            CarbonLoadOptionConstants.CARBON_OPTIONS_BINARY_DECODER,
+            CarbonLoadOptionConstants.CARBON_OPTIONS_BINARY_DECODER_DEFAULT);
+      }
+      // check and load binary decoder
+      carbonLoadModel.setBinaryDecoder(binaryDecoder);
+    } catch (Exception e) {

```

Comment:
Why to add a try catch, when no exception is thrown from this block of code

Reply:
only follow validateAndSetColumnCompressor, validateGlobalSortPartitions and so on.

Reply:
Please remove